src/amici.cpp

```diff
@@ -80,19 +80,19 @@ std::unique_ptr<ReturnData> runAmiciSimulation(Solver &solver, const ExpData *ed
     } catch (amici::IntegrationFailure& ex) {
         rdata->invalidate(ex.time);
         rdata->status = ex.error_code;
-        amici::errMsgIdAndTxt("AMICI:mex:simulation","AMICI simulation failed at t = %f:\n%s\nError occured in:\n%s",ex.time,ex.what(),ex.getBacktrace());
+        amici::warnMsgIdAndTxt("AMICI:mex:simulation","AMICI simulation failed at t = %f:\n%s\nError occured in:\n%s",ex.time,ex.what(),ex.getBacktrace());
```

Comment:
warnings allows us to still return rdata, which is nice because you can actually inspect the simulation results to get some idea where things went wrong.